README.md

```diff
@@ -127,27 +126,77 @@ const myStamp = compose(config, warnOnCollisions);
 ```
 
 
-### Stamp `options` Reserved Keys
+### Stamp Options
 
-The following are reserved keys for the stamp options object:
+It is recommended that stamp initializers only use one argument: The stamp `options` argument. There are no reserved properties and no special meaning. However, using multiple arguments for a stamp could create conflicts where multiple stamps expect the same argument to mean different things. Using named parameters, it's possible for stamp creator to resolve conflicts with `options` namespacing. For example, if you want to compose a database connection stamp with a message queue stamp:
```

Comment:
>  However, using multiple arguments for a stamp could create conflicts where multiple stamps expect the same argument to mean different things.

When you say multiple arguments, do you mean multiple properties in the `options` argument?